ethcore/wasm/src/env.rs

```diff
@@ -207,6 +213,7 @@ pub struct ImportResolver {
 	memory: RefCell<Option<MemoryRef>>,
 
 	have_create2: bool,
+	have_gasleft: bool,
```

Comment:
better to rename this to `has_gasleft` and the one above to `has_create2` for consistency

Reply:
shall we rename everything else in the stack https://github.com/paritytech/parity-ethereum/blob/108590d924bcc00e62b894de02516ffb2378be11/ethcore/src/spec/spec.rs#L177 ?

Reply:
oh, i didn't realized

probably not :)